src/ty/mod.rs

```diff
@@ -197,6 +204,9 @@ pub enum TypeDef<T: Form = MetaForm> {
     Compact(TypeDefCompact<T>),
     /// A PhantomData type.
     Phantom(TypeDefPhantom<T>),
+    #[cfg(feature = "bit-vec")]
+    /// A BitVec type.
+    BitVec(TypeDefBitVec<T>),
```

Comment:
I think dtolnay mentioned that for compatibility with respect to different crate feature sets it is better to replace the inner type of `BitVec` with a placeholder type (dummy) of `not(feature = "bit-vec")` instead of entirely removing this variant.

Reply:
That is much better, will avoid issues with incompatible decoding if we add variants after this.

Reply:
I've removed the feature flag from this variant now so it will be binary compatible, and also so that decoding/deserializing is supported even without the feature enabled.